blocks/measuring-impact-cards/measuring-impact-cards.js

```diff
@@ -0,0 +1,66 @@
+import { div, a, p } from '../../scripts/dom-helpers.js';
+import { moveInstrumentation, fetchLanguagePlaceholders } from '../../scripts/scripts.js';
+
+const getShortendIndicatorName = (indicatorTag, placeholdersData) => {
+  // Return Same indicator tag content if hyphen is not present
+  if (indicatorTag && indicatorTag.indexOf('-') < 0) return indicatorTag;
+
+  const indicatorKey = indicatorTag.split('-')[0].trim().replace(/[^a-zA-Z0-9]/g, '');
+  return placeholdersData[indicatorKey] ? placeholdersData[indicatorKey] : indicatorKey;
+};
+
+const getColorCode = (indicatorTag, placeholdersData) => {
+  if (indicatorTag && indicatorTag.indexOf('-') < 0) return '#07AB50;';
+  const indicatorColorKey = indicatorTag.split('-')[1].trim().replace(/[^a-zA-Z0-9]/g, '');
+  return placeholdersData[indicatorColorKey] ? placeholdersData[indicatorColorKey]
```

Comment:
@singhpritam18  use prefix in keys to add context of block

Reply:
What's the expectation here ?
 
do you mean to have color keys like verticalname-color e.g. planet-green.
If yes, idea behind to have vertical color code authorable, is to give flexibility to add any color for the vertical. If we add color code to keys then wouldn't be confusing for authors later on ?

Reply:
Incorporated as we discussed over call to have block name as prefix for Shortend Tag key and 'theme' as prefix for Verticals